### ISSUE TITLE
Mana basics quest not recognizing the wand when crafted.

### ISSUE THREAD

**Issue by Teboran** [TeamAOF/All-of-Fabric-7#106]
Botania quest involving the wand didn't recognize crafting the wand. I manually adjusted the config for the quest (mana basics) and got it working on my server. My "fix" updated the quest for anyone the had the wand.

Added under title: "Craft a wand"

   "item": "botania:twig_wand",
      "amount": 1,
      "collection": "AUTOMATIC",

**Comment by natereprogle** [comment]
This didn't quite work for me, but it put me on the correct path. Within your game files under `/config`, there is a folder called `heracles`. Within that folder is all the quest definitions for the modpack. Navigate to `quests/botania` and open the "mana basics.json" file. The "wand of the forest" quest is a crafting quest, but I changed it to an acquire quest. Here is what mine looked like after editing. I essentially took out the `recipes: []` array and added the item, amount, and collection tags as noted above. Basically, I copied the "mana pool" quest but changed it to work for `botania:twig_wand`. For anyone wanting to be able to copy and paste it, I pasted my changes below.

```json
"wand of the forest": {
      "type": "heracles:item",
      "item": "botania:twig_wand",
      "amount": 1,
      "collection": "AUTOMATIC",
      "title": "Craft a Wand",
      "icon": {
        "item": {
          "id": "minecraft:air",
          "count": 0
        },
        "type": "heracles:item"
      }
    },
```
